crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -32,10 +34,15 @@ pub type StaticCommittee<T> = GeneralStaticCommittee<T, BLSPubKey>;
 impl<T, PUBKEY: SignatureKey> GeneralStaticCommittee<T, PUBKEY> {
     /// Creates a new dummy elector
     #[must_use]
-    pub fn new(_nodes: &[PUBKEY], nodes_with_stake: Vec<PUBKEY::StakeTableEntry>) -> Self {
+    pub fn new(
+        _nodes: &[PUBKEY],
+        nodes_with_stake: Vec<PUBKEY::StakeTableEntry>,
+        nodes_without_stake: Option<Vec<PUBKEY>>,
```

Comment:
I have no context of course, but I'm curious about the use of `Option` here. If it's a `Vec`, would `vec![]` or something similar not be a good default case?

Reply:
Sorry I think you reviewed the old version.

Reply:
Hmm, when I refresh I still see the Option type being used. Are you saying you have new commits coming?

Reply:
It is already there.

Reply:
Sorry, let me ask a different way. My original question was asking if the `Option` type was necessary here, since an empty Vec type might be sufficient in this case to imply a value is not present. I was curious about your thought process for adding it in this case.

Reply:
Yeah, I think empty vec covers that case, I just went with the flow.